package.json

```diff
@@ -17,6 +17,7 @@
   "schema": "3.1.4",
   "templateVersion": "2.0.0",
   "dependencies": {
+    "@formio/premium": "^1.15.1-rc.9",
```

Comment:
We cannot include this as dependency. This is an Open Source library and access to "@formio/premium" is not available to the public.